packages/devtools/lib/src/inspector/inspector_controller.dart

```diff
@@ -178,14 +179,14 @@ class InspectorController implements InspectorServiceClient {
   }
 
   void endShowNode() {
-    highlightShowNode(null);
+    highlightShowNode(null, null);
   }
 
   bool highlightShowFromNodeInstanceRef(InspectorInstanceRef ref) {
-    return highlightShowNode(valueToInspectorTreeNode[ref]);
+    return highlightShowNode(valueToInspectorTreeNode[ref], null);
   }
 
-  bool highlightShowNode(InspectorTreeNode node) {
+  bool highlightShowNode(InspectorTreeNode node, Icon icon) {
```

Comment:
nit: maybe make one or both of these params named and required?

Reply:
done

Reply:
sg